Odata-docs/webapi-8/overview/overview.md

```diff
@@ -0,0 +1,54 @@
+- Handles serialization and deserialization of requests and responses
+- Validation of requests and responses based on the types defined in the OData model
+- Support for batch requests, which allow you to execute multiple operations in a single API request
+- Handles [OData query options](/odata/concepts/queryoptions-overview) out of the box by transforming queries to your LINQ-enabled data source:
+  - `$filter` allows you to filter results based on specific conditions: `/Products?$filter=Price lt 1000 and Category eq 'Electronics'` will only return electronic products whose price is less than 1,000.
+  - `$select` allows you to limit the fields that are included in the response: `/Products?$select=Name,Category,DateCreated` will only include the `Name`, `Category`, `DateCreated` fields in the response
+  - `$expand` allows you to join and include related data in the response: `/OrderItems?$expand=Product` will return order items and include the related product in the response
+  - `$skip` and `$top` allow you to limit the number of records in the response and can be used for client-driven pagination: `/Products$top=10&$skip=20&$orderby=Name` returns the third page of 10 products sorted by name.
+  - `$apply` allows you to apply aggregations: `/Orders?$apply=groupby(Product/Name, aggregate($count as Count))` returns the number of orders per product name.
+
+
+## Why choose OData Web API:
+
+Here are some of the benefits and common use cases for using OData Web API:
+
+- Creating a standards-compliant Web API with support for filtering, selecting, expanding among other capabilities.
+- Interoperability with OpenAPI and Swagger
+- Type-safety
+- Ability to generate client code from OData model description
+- Expose a REST-API layer to your data source for analytics functions.
+- Add advanced querying capabilities via OData query options to your REST API even if it's not based on OData.
+
+## NuGet packages
+
+The library is shipped on NuGet under the package name: `Microsoft.AspNetCore.OData`.
+
+We also ship a library separately for building OData models from .NET model classes. This is available under the package name: `Microsoft.OData.ModelBuilder`.
+
+## Choosing the right version
+
+There are two major versions of OData Web API under active maintenance and support:
+- OData Web API 8: `Microsoft.AspNetCore.OData` 8.x
+- OData Web API 7: `Microsoft.AspNetCore.OData` 7.x and `Microsoft.AspNet.OData` 7.x
+
+For newer development of OData services on .NET, OData Web API 8 is recommended. However, OData Web API 7 is actively maintained and some newer features in OData Web API 8 are back-ported to OData Web API 7 where possible. There are also features in Web API 7 that have not been ported to Web API 8, e.g., [automatic support for deeply nested paths](/odata/webapi/automatic-nested-paths-with-enable-nested-paths).
```

Comment:
Do we have a page with a list of features that are not back-ported?If we do we can include that list instead of a single example.  if not we may want to remove the sentence about features that are not back ported for now until we can link to a list of these features

Reply:
We don't have such a page in the official docs. I can create a list here, but it won't be exhaustive (since I don't know the full extent of what's not been ported to 8 or backported to 7. Maybe others can contribute.

Reply:
I've gotten one example from a @KenitoInc  of a feature that's not back-ported from 8 to 7 (i.e. it's only available to 8), but I don't have an exhaustive list. Not sure if you'd prefer that I mention that example or still remove the sentence for now?

Reply:
Actually, after re-reading the statement and your comment, I have opted to remove it for now.